src/algorithm/butterflies.rs

```diff
@@ -253,19 +283,18 @@ impl<T: FftNum> Butterfly4<T> {
         }
     }
     #[inline(always)]
-    pub(crate) unsafe fn perform_fft_contiguous(
+    pub(crate) unsafe fn perform_fft_butterfly<U: LoadStore<Complex<T>> + ?Sized>(
```

Comment:
I see this "+?sized" bound peppered everywhere. Skimming through, it doesn't look like there's anywhere that we *don't* add this - is there any way to bake this into the LoadStore trait definition?

Reply:
I tried adding it to add it originally, but it turns out that you can't have any ?Trait in a supertrait. I was able to resolve the issue by switching to use "U" from "&mut U", adding "DerefMut" as a bound on "U", and making LoadStore<T> into a supertrait where U: LoadStoreFns<Complex<T>> + DerefMut.